src/extension.ts

```diff
@@ -42,7 +42,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	context.subscriptions.push(vscode.commands.registerCommand(extensionFunctions.SCAFFOLD_PROJECT_COMMAND, extensionFunctions.scaffoldProjectFromJson));
 	context.subscriptions.push(vscode.commands.registerCommand(extensionFunctions.CREATE_WORKSPACE_FOLDER_COMMAND, extensionFunctions.createTemporaryFolderAsWorkspaceRoot));
 	context.subscriptions.push(vscode.commands.registerCommand(extensionFunctions.OPEN_TUTORIAL_COMMAND, extensionFunctions.openDidactWithDefault));
-	context.subscriptions.push(vscode.commands.registerCommand(extensionFunctions.START_DIDACT_COMMAND, extensionFunctions.startDidact));
+	context.subscriptions.push(vscode.commands.registerCommand(extensionFunctions.START_DIDACT_COMMAND, extensionFunctions.findOrCreateDidact));
```

Comment:
It does the expected behavior from the user's perspective. If the didact file is already open, we don't want to open it again. If it's not open, we want to open it. Either way, it "starts" the didact file without completely changing the way that users currently think of "starting" a didact file.

Reply:
When I read the `findOrCreateDidact`, I do not understand the behavior that you explained.
I understand "create Didact"  as create a Didact file, a *.didact.adoc or *.didact.md; not create a rendering of the Didact file inside a webview.

Reply:
> i think that it needs to be documented as a limitation or implemented

@apupier I'm curious where you think it should be documented -- but even more interested in how you think it might be implemented. At this point, it is not possible to automatically have an id applied to every heading.

Reply:
> When I read the `findOrCreateDidact`, I do not understand the behavior that you explained.

@apupier what would you prefer? `findOrStartDidact`?

Reply:
based on implementation:
```
	if (uri) {
		const parentPanel = didactManager.getByUri(uri);
		if (parentPanel) {
			parentPanel._panel?.reveal();
		} else {
			await startDidact(uri, viewColumn);
		}
	}
```
Maybe `revealOrStartDidact` , `revealOrStartDidactTutorial`

Reply:
went with `revealOrStartDidactByURI`